S-L-Murphey/Level-Up-API
# Chapter 7: Listing Games

#### Changes Made

1. Modified file `levelup/urls.py` and `levelupapi/fixtures/games.json` to include listing games functionality

#### Steps to Review

1. Checkout this branch locally.
   ```
   git fetch --all
   git checkout slm-levelup-7
   ```
2. Ensure the backend and client are both running.
4. Test app functionality.
   > Example: When user logins, all games should display with the name, maker, description, and the number of players needed.

5. View code file.
   > Confirm file modifications are present as indicated above.
   > Confirm no unused code or extraneous comments exist.
